releases/release-1.21/release-notes/maps/pr-98750-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 98750
+releasenote:
+  text: Fixes problem where invalid selector on PDBs leads to a nil pointer dereference
```

Comment:
nit: ideally we should elaborate on what `PDB` is :smile:

Reply:
Should be clearer now.